### PR TITLE
Create menu and logo

### PR DESCRIPTION
**Pull request by jinety**

### CODE REVIEW

File: practice-one/src/practice-one.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+.nav{
+    align-items: flex-start;
+    display: flex;
+    height: 43px;
+    width: 1328px;
+    margin-top: 40px;
+
```
Comment (phuocduong-agilityio) [inline practice-one/src/practice-one.css]: Remove empty space here

File: practice-one/src/practice-one.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    top: 40px;
+}
+.open-enterprise{
+    font-family: 'Roboto', sans-serif;
+    letter-spacing: -0.04em;
+    margin-left: 125px;
+    height: 38px;
+    max-width: 400px;
+    font-weight: 400;
+    font-size: 32px;
```
Comment (phuocduong-agilityio) [inline practice-one/src/practice-one.css]: Collect font-size as variable, the same for color

File: practice-one/src/practice-one.css [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    margin-top: 30px;
+}
+.logo{
+    position: absolute;
```
Comment (phuocduong-agilityio) [inline practice-one/src/practice-one.css]: Could you explain why we need position absolute?

File: practice-one/src/practice-one.css [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    left: 56px;
+    height: 40px;
+    width: 68px;
+    margin-top: 30px;
+    float: left;
+  
+}
+.nav-link{
+    float: right;
```
Comment (phuocduong-agilityio) [inline practice-one/src/practice-one.css]: Use another way instead of float. Float can break application

File: practice-one/src/practice-one.css [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+    list-style: none;
+    margin-top: 30px;
+    margin-right: -1400px;
```
Comment (phuocduong-agilityio) [inline practice-one/src/practice-one.css]: This is hard code???

File: practice-one/src/practice-one.html [hunk 6]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="practice-one.css">
```
Comment (phuocduong-agilityio) [inline practice-one/src/practice-one.html]: Style file should be style.css

File: practice-one/src/practice-one.html [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+        <title>practice-one</title>
+    </head>
+    <body>
+        <div class="nav">
+            <div class="frame">
+                <img src="../img/logo.png" class="logo" alt="">
```
Comment (phuocduong-agilityio) [inline practice-one/src/practice-one.html]: Missing alt content

File: practice-one/src/practice-one.html [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+                <div class="open-enterprise">
+                    <span>Open Enterprise</span>
+                </div>
+            </div>
+            <div>
```
Comment (phuocduong-agilityio) [inline practice-one/src/practice-one.html]: Why need div here?

File: practice-one/src/practice-one.html [hunk 9]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="practice-one.css">
+        <title>practice-one</title>
+    </head>
+    <body>
+        <div class="nav">
```
Comment (phuocduong-agilityio) [inline practice-one/src/practice-one.html]: Consider header tag

File: practice-one/src/practice-one.css [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0;
+    position: absolute;
+    width: 500px;
+    height: 40px;
+    left: 0px;
+    top: 40px;
+}
```
Comment (phuocduong-agilityio) [inline practice-one/src/practice-one.css]: Add empty space between 2 styles

File: practice-one/src/practice-one.css [hunk 11]
```diff
@@ -0,0 +1,60 @@
...
+    float: right;
+    list-style: none;
+    margin-top: 30px;
+    margin-right: -1400px;
+}
+.nav-link li{
+    display: inline-block;
+    margin-right: 30px;
+}
+.nav-link li a{
```
Comment (phuocduong-agilityio) [inline practice-one/src/practice-one.css]: Avoid style in element tags, should add class for styling

File: practice-one/src/practice-one.html (hunk 8 above)
Comment (jinety) [reply practice-one/src/practice-one.html]: To create a separate block that does not match the tags above

File: practice-one/src/practice-one.css (hunk 11 above)
Comment (jinety) [reply practice-one/src/practice-one.css]: fixed

File: practice-one/src/practice-one.css (hunk 1 above)
Comment (jinety) [reply practice-one/src/practice-one.css]: fixed

File: practice-one/src/practice-one.css (hunk 2 above)
Comment (jinety) [reply practice-one/src/practice-one.css]: fixed

File: practice-one/src/practice-one.html (hunk 6 above)
Comment (jinety) [reply practice-one/src/practice-one.html]: fixed

File: practice-one/src/practice-one.html (hunk 7 above)
Comment (jinety) [reply practice-one/src/practice-one.html]: fixed

File: practice-one/src/practice-one.html (hunk 9 above)
Comment (jinety) [reply practice-one/src/practice-one.html]: fixed

File: practice-one/src/practice-one.css (hunk 4 above)
Comment (jinety) [reply practice-one/src/practice-one.css]: fixed

File: practice-one/src/practice-one.css (hunk 5 above)
Comment (jinety) [reply practice-one/src/practice-one.css]: fixed

File: practice-one/src/practice-one.css (hunk 3 above)
Comment (jinety) [reply practice-one/src/practice-one.css]: To fix the position of the logo

File: practice-one/src/style.css [hunk 12]
```diff
@@ -0,0 +1,59 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+:root{
+    --font-size-m: 32px;
+    --font-size-s: 20px;
+    --font-weight: 400; 
+    --text-color: black;
+    --text-color-secondary: #71A894;
+    --font-family: 'Roboto', sans-serif;
+}
+.nav{
```
Comment (phuocduong-agilityio) [inline practice-one/src/style.css]: Add space between .nav and {. Pls check around

File: practice-one/src/index.html [hunk 13]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="style.css">
+        <title>Open Enterprise</title>
+    </head>
+    <body>
+        <div class="nav">
+            <div class="logo-and-company-name">
```
Comment (phuocduong-agilityio) [inline practice-one/src/index.html]: Should use another class name. Ex: Logo or branch identify, etc

File: practice-one/src/index.html [hunk 14]
```diff
@@ -0,0 +1,22 @@
...
+                <img src="../img/logo.png" class="logo" alt="logo">
+                <div class="open-enterprise">
```
Comment (phuocduong-agilityio) [inline practice-one/src/index.html]: Should NOT use `open-enterprise` (company name) as a classname

File: practice-one/src/style.css [hunk 15]
```diff
@@ -0,0 +1,59 @@
...
+    display: flex;
+    align-items: flex-start;
+}
+.logo-and-company-name{
+   flex: 45%;
+   display: flex;
+   flex-direction: row;
+   align-items: center;
+   padding: 0;
+   left: 0px;
```
Comment (phuocduong-agilityio) [inline practice-one/src/style.css]: Do we need left and top style here?

**Pull request merged**